app/backend/src/clj/teet/activity/activity_db.clj

```diff
@@ -181,3 +182,20 @@
        (-> (du/entity db activity-id)
            :activity/name :db/ident)
        #{}))
+
+(defn activity-task-has-files?
+  "Check a task under activity has undeleted files."
+  [db activity-id]
+  (seq
```

Comment:
This returns a list `(true true true)` for example not a boolean. Might be worth it to refactor it so it returns just a boolean.

Reply:
Also this whole function could be more readable if it's done using the threading macro 

`(-> tasks (comp :db/id first) .... etc)`